modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -22,12 +22,12 @@ import {fetch, navigate, openModal, openToast} from 'frontend-js-web';
 import React, {useEffect, useRef, useState} from 'react';
 
 import {API_URL, OBJECT_RELATIONSHIP} from '../Constants';
-import {FDSViewSectionInterface} from '../FDSView';
+import {IFDSViewSectionInterface} from '../FDSView';
```

Comment:
👍  I haven't look too closely at names for the time being since we are sort of defining features as we go, perhaps once we get the bulk of features out we can refactor and rename things to be a bit more cohesive.